src/test/java/edu/hm/hafner/analysis/IssuesTest.java

```diff
@@ -0,0 +1,323 @@
+package edu.hm.hafner.analysis;
+
+import java.util.SortedSet;
+import java.util.TreeSet;
+import java.util.UUID;
+
+import org.junit.jupiter.api.Test;
+
+import static edu.hm.hafner.analysis.assertions.IssuesAssert.*;
+import static edu.hm.hafner.analysis.assertions.IssueAssert.*;
+import static org.assertj.core.api.Assertions.*;
+import edu.hm.hafner.util.NoSuchElementException;
+
+/**
+ * Testing Class for Issues.
+ *
+ * @author Mark Tripolt
+ */
+class IssuesTest {
+
+    @Test
+    void addIssueToIssues() {
+        Issues issues = new Issues();
+        Issue issue = new IssueBuilder().build();
+        issues.add(issue);
+
+        assertThat(issues)
+                .hasSize(1)
+                .hasHighPrioritySize(0)
+                .hasLowPrioritySize(0)
+                .hasNormalPrioritySize(1)
+                .hasIssueOnIndex(issue, 0)
+                .hasNumberOfFiles(1)
+                .hasToString(String.format("%d issues", issues.size()));
+
+        assertThat(issues.get(0))
+                .hasFileName("-")
```

Comment:
Das wurde schon in IssueTest getestet: es genügt `assertThat(...).isSameAs(issue)`.